cli/src/runner.rs

```diff
@@ -179,54 +202,66 @@ pub async fn run_quarantine(
     repo: &BundleRepo,
     delay: std::iter::Take<ExponentialBackoff>,
 ) -> anyhow::Result<QuarantineRunResult> {
-    let quarantine_results = if run_result.failures.is_empty() {
-        QuarantineBulkTestStatus {
-            group_is_quarantined: false,
-            quarantine_results: Vec::new(),
-        }
-    } else {
+    let mut quarantine_results = QuarantineBulkTestStatus {
+        group_is_quarantined: false,
+        quarantine_results: Vec::new(),
+    };
```

Comment:
nit: can we define this below, where we use/mutate it?

nit: Also, we should add a `#[derive(Default)]` to `QuarantineBulkTestStatus` that way we can simply do the following

```suggestion
    let mut quarantine_results = QuarantineBulkTestStatus::default();
```